case-lib/lib.sh

```diff
@@ -1133,34 +1214,21 @@ set_alsa_settings()
 
 reset_sof_volume()
 {
+    level_db=$(is_ipc4 && echo "100%" || echo "0dB")
```

Comment:
```suggestion
    level_db=$(if is_ipc4; then echo "100%"; else echo "0dB"; fi)
```

`A && B || C` is a very dangerous construct and bad example. There is no bug in this particular case but there very easily can be, compare:

```shell
if true; then grep notfound /etc/passwd ; else 'ELSE is not run'; fi
true && grep notfound /etc/passwd || echo 'ELSE is also run!!'
```

Reply:
Thank you for that hint.